package.json

```diff
@@ -26,6 +26,7 @@
     "concurrently": "^5.3.0"
   },
   "dependencies": {
-    "dotenv": "^8.2.0"
+    "dotenv": "^8.2.0",
+    "react-loader-spinner": "^3.1.14"
```

Comment:
this need to be installed in the public project not he root project

Reply:
Fixed this